geth/peers/topicpool.go

```diff
@@ -154,12 +149,12 @@ func (t *TopicPool) ConfirmAdded(server *p2p.Server, nodeID discover.NodeID) {
 
 	// move peer from pool to connected peers
 	t.movePeerFromPoolToConnected(discV5NodeID)
-	// make sure `dismissed` is restarted
+	// make sure `dismissed` is reset
 	peer.dismissed = false
 
 	// when the lower limit is reached, we can switch to slow mode
```

Comment:
we're not checking lower limit reached anymore on this conditional, so this comment should be updated